src/quo2/components/qr_code/qr.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.qr-code.qr
+  (:require [react-native.core :as rn]))
+
+(defn user-profile-qr-code
+  [{:keys [key-uid public-key port qr-size]}]
+  (let [profile-qr-url         (str "https://join.status.im/u/" public-key)
+        base-64-qr-url         (js/btoa profile-qr-url)
+        profile-image-type     "large"
+        error-correction-level 4
+        superimpose-profile?   true
+        media-server-url       (str "https://localhost:"
```

Comment:
same here, we should pass full url to component

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/6dd9897540ca2ab37b0d190dfae3cd80ddb1b56d